### ISSUE TITLE
Searx instances throwing "searchcode doc" error when using certain search terms/phrase

### ISSUE THREAD

**Issue by DaSkyburner2020** [searx/searx#2073]
Hey just found a strange error:
When searching for the phrase "ews cheat sheet" with a searx instance and then switching to the "IT" tab the instance/website throws "searchcode doc" errors. 
Confirmed and reproducable with Google Chrome (build 83.0.4103.116, active addons: Ghostery, ublock origin, adblock plus), Firefox (build: 78.0.2, active addons: Ghostery, adblock plus (also tested without addons)), Vivaldi (build: 3.1.1929.45, active addons: Ghostery, ublock origin (also tested without addons)).
Confirmed and reproduced on Linux (Manjaro, kernel: 5.4.44_rt26-1) and Windows 10 (1909).

Example error codes:

> lobste.rs (unexpected crash Empty url resultset), searchcode doc (unexpected crash Extra data: line 1 column 5 (char 4))

> arch linux wiki (unexpected crash can't concat str to bytes), searchcode doc (unexpected crash Extra data: line 1 column 5 (char 4)), lobste.rs (timeout)

> searchcode doc (unexpected crash Extra data: line 1 column 5 (char 4)), lobste.rs (timeout)

> searchcode doc (unexpected crash Extra data: line 1 column 5 (char 4))

Tested on like 10 instances but I haven't documented which.
If you need any further info, just drop me a message.
Hope this report helps :)

**Comment by unixfox** [comment]
tldr: the searchcode_doc engine is broken.
I can replicate the bug in a local Searx instance with any keyword.

**Comment by dalf** [comment]
The engine has been removed: https://github.com/searx/searx/pull/2372
